AirCasting/Graph/GraphDataGenerator.swift

```diff
@@ -0,0 +1,43 @@
+// Created by Lunar on 01/06/2021.
+//
+
+import Foundation
+
+class GraphDataGeneraator {
```

Comment:
final, double aa

Reply:
There's no GraphDataGenerator anymore. I don't know why github didn't mark this comment as outdated.

Reply:
probably just removed form Xcode reference and not from file system.